pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -68,6 +74,10 @@ type VSphere struct {
 	vsphereInstanceMap map[string]*VSphereInstance
 	// Responsible for managing discovery of k8s node, their location etc.
 	nodeManager *NodeManager
+
+	clientBuilder controller.ControllerClientBuilder
```

Comment:
There is no need for these fields to be part of VSphere struct as these are used locally only in Initialize function.